### DISCUSSION TITLE
G29 P3 is this enabled for UBL

### DISCUSSION THREAD

**Discussion by jxer** [Jyers/Marlin#791] [General Discussion]
Is this enabled? i tried to find it in source code, but i am very new to this. seems a lot of settings are all over the place.

**Comment by Tuncay-Ayhan** [comment]
> Is this enabled? i tried to find it in source code, but i am very new to this. seems a lot of settings are all over the place.

Here you go: https://github.com/Jyers/Marlin/discussions/240#discussioncomment-471747
